### PR TITLE
index.js - a basic solution to support testing URLs that contain commas.

### PR DESCRIPTION
**Pull request by mykola-rykov**
A basic solution to support testing URLs that contain commas.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -153,6 +153,20 @@ if (okToStart) {
...
+// Only the last value - URL - may contain commas.
+// We join all items after two first if the array of data contains more then 3 items.
+function getUrl(page) {
+  let url;
+  const pageParts = page.split(',');
+  if (pageParts.length > 3) {
+	url = pageParts.slice(2, pageParts.length).join();
+  } else {
+   url = pageParts[2];
+  }
```
Comment (samdutton) [inline src/index.js]: Nice!

Even simpler might just be to get `pageParts.slice(2, pageParts.length).join();` for every page.

Don't think that will affect performance.

In other words, in `audit()`:

```
const pageParts = page.split(',');
// First two pageParts are website name and page name. 
// URL may contain commas.
const url = pageParts.slice(2, pageParts.length).join();
```
(Or put this in a `getURL(page)` function.)

**Comment by samdutton** [comment]
Thank you! Embarrassed that I didn't think about this...

Added a suggestion in a comment.

File: src/index.js (hunk 1 above)
Comment (mykola-rykov) [reply src/index.js]: Great, thank you! Made a change to the `getURL(page)` function as you reccommended.

**Comment by samdutton** [comment]
Thanks again — merging.

**Pull request merged**